tfs/src/main/java/hudson/plugins/tfs/model/AbstractHookEvent.java

```diff
@@ -275,15 +277,32 @@ private Boolean branchMatches(final GitCodePushedEventArgs gitCodePushedEventArg
                     continue;
                 }
 
+                this.envVars = project instanceof Job ? buildEnv((Job) project) : new EnvVars();

```

Comment:
seems like the check for instance of job could be pushed down into buildEnv. Not a big deal, just makes this code more readable.

Reply:
Good suggestion. I considered that before, but it needed to change buildEnv to take "project" type instead "Job" type. As its internal logic is only applicable for a Job, the input type of "Job" seemed more appropriate. Luckily it's just one line of difference and doesn't look too bad with current approach. :)